interfaces/builtin/custom_device.go

```diff
@@ -408,13 +416,16 @@ func (iface *customDeviceInterface) UDevConnectedPlug(spec *udev.Specification,
 	// Create a map in which will store udev rules indexed by device name
 	deviceRules := make(map[string]string, len(allDevicePaths))
 
-	// Generate a basic udev rule for each device; we put them into a map
-	// indexed by the device name, so that we can overwrite the entry later
-	// with a more specific rule.
+	placeholderRule := "###PLACEHOLDER###"
```

Comment:
nitpicks: this can be a const,  also the format make it look like it's going to be string replaced which is not the case,  maybe just "\<placeholder>" ?